.github/workflows/ci.yml

```diff
@@ -0,0 +1,65 @@
+name: CI Build Status
+
+on: [push, pull_request]
+
+env:
+  RUST_BACKTRACE: 1
+  # We install a known-to-have-rustfmt version of the nightly toolchain
+  # in order to run the nightly version of rustfmt, which supports rules
+  # that we depend upon. When updating, pick a suitable nightly version
+  # from https://rust-lang.github.io/rustup-components-history/
+  # See .rustfmt.toml for the list of unstable features.
+  KNOWN_TO_HAVE_RUSTFMT: nightly-2020-07-12
```

Comment:
Pinning to the existing old version from .travis.yml for now since newer version causes quite a lot of dead code warnings which I don't feel comfortable to fix here.